README.md

```diff
@@ -294,7 +295,29 @@ For **∅ or more** and **1 or more** element(s) of the same type you can use th
 Union types are denoted with the pipe symbol, `|`:
 
 ```js
-(userInput: String|Number) => String|Number
+(userInput: String | Number) => String | Number
+```
+
+### Negated Types
+
+It is sometime easier and more informative to delimit a type by defining what it's not. The negation operator let's you exclude by substracting from `Any`.
+
+```js
+JSON::parse(String, reviver: Function)
+  => Boolean | Number | String | Object | Array | null,
+  throws SyntaxError
+
+// is less concise than
+
+JSON::parse(String, reviver: Function)
+  => !Function & !Void & !Symbol,
```

Comment:
2 caveats:

1. is Object may have different interpretations
2. it's highly unlikely but a new builtin might be introduced (e.g. Symbol was)